bin/sl-deploy.js

```diff
@@ -12,11 +13,13 @@ var fs = require('fs');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+if (path.basename(process.cwd()) === 'test')
```

Comment:
why do we use a different root dir when running tests? also, you cannot tell from the cwd if the tests are running (`node test/test-x.js` is allowed, and is how many of run the tests during dev workflow)